cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -175,7 +174,8 @@ newtype TxId = TxId (Shelley.Hash StandardCrypto Shelley.EraIndependentTxBody)
   deriving newtype (IsString)
                -- We use the Shelley representation and convert the Byron one
 
-deriving newtype instance ToJSON TxId
+instance ToJSON TxId where
+  toJSON = Aeson.String . Text.decodeUtf8 . serialiseToRawBytesHex
```

Comment:
Perhaps we should add `Text.decodeUtf8 . serialiseToRawBytesHex` to `Cardano.Api.SerialiseRaw`, e.g.
```
serialiseToRawBytesHexText = Text.decodeUtf8 . serialiseToRawBytesHex
```